### PR TITLE
Add bulk insert events API

### PR DESCRIPTION
**Pull request by erikjohnston**

### CODE REVIEW

File: synapse/storage/events.py [hunk 1]
```diff
@@ -293,19 +366,19 @@ def _persist_event_txn(self, txn, event, context, backfilled,
...
-        for hash_alg, hash_base64 in event.hashes.items():
-            hash_bytes = decode_base64(hash_base64)
-            self._store_event_content_hash_txn(
-                txn, event.event_id, hash_alg, hash_bytes,
-            )
+        # for hash_alg, hash_base64 in event.hashes.items():
+        #     hash_bytes = decode_base64(hash_base64)
+        #     self._store_event_content_hash_txn(
+        #         txn, event.event_id, hash_alg, hash_bytes,
+        #     )
```
Comment (NegativeMjark) [inline synapse/storage/events.py]: Why are these commented out?

Comment (erikjohnston) [inline synapse/storage/events.py]: Ooooh, an interesting question. I don't think those tables actually get read from currently? So the question is whether we still want to keep them?

Comment (NegativeMjark) [inline synapse/storage/events.py]: If they aren't being read from I guess we should remove them. We can always add them back if they turn out to be necessary.

**Comment by NegativeMjark** [comment]
LGTM

**Pull request merged**